include/zephyr/drivers/dac.h

```diff
@@ -28,17 +28,16 @@ extern "C" {
 /**
  * @struct dac_channel_cfg
```

Comment:
```suggestion
```

Redundant, as docstring is adjacent to the struct

Reply:
+1, thx. I guess my thinking was to keep the diff minimal but cleaning up doesn't hurt indeed.
Fixed!